modules/end-user-guide/partials/ref_options-for-che-editor-yaml.adoc

```diff
@@ -1,12 +1,37 @@
-[id="using-an-embedded-editor-ddefinition-for-your-ide_{context}"]
-= Using an embedded editor definition for your IDE
+[id="options-for-che-editor-yaml_{context}"]
+= Options for che-editor.yaml
 
-If you have specific requirements for your project that aren't addressed by standard IDE behaviour, you can customize the project IDE by using the `inline` directive to put a complete IDE definition in the `che-editor.yaml` file.
+The `che-editor.yaml` file supports optional parameters to suit the needs of more complex projects:
```

Comment:
Reading your sentence I understand that `id` is mandatory and `reference` and `inline` are optionals. But those are alternatives to `id`.

```suggestion
The `che-editor.yaml` file supports using a `reference` to a remote IDE definition or an `inline` IDE definition as an alternative to the `id` parameter. This is useful when the IDE is not defined in the plugin registry:

.`reference` points to a remote `che-editor.yaml` file
====
[source,yaml]
----
reference: https://gist.github.com/.../che-editor.yaml # required
----
====

.`inline` specifies a complete IDE definition when your project requires a customized IDE
====
[source,yaml]
----
inline:
  components:
  (...)
  events:
  (...)
   commands:
  (...)
----
====

The `che-editor.yaml` file supports optional parameters to suit the needs of more complex scenarios: `regisryUrl` and `override`:
```